apps/core/src/hooks/useGetClockTimestamp.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { useIotaClient } from '@iota/dapp-kit';
+import { IotaClient } from '@iota/iota-sdk/client';
+import { useQuery } from '@tanstack/react-query';
+
+const CLOCK_TIMESTAMP_PACKAGE_ID = '0x06';
```

Comment:
I would rename this to:
```suggestion
const CLOCK_PACKAGE_ID = '0x06';
```